examples/src/mysql_demo/mod.rs

```diff
@@ -144,9 +145,10 @@ pub(crate) async fn render_cli(args: CliArgs) -> Result<()> {
     if cli.memory_db {
         let db = akd::storage::memory::AsyncInMemoryDatabase::new();
         let storage_manager = StorageManager::new_no_cache(db);
-        let mut directory = Directory::<TC, _, _>::new(storage_manager, vrf)
-            .await
-            .unwrap();
+        let mut directory =
+            Directory::<TC, _, _>::new(storage_manager, vrf, AzksParallelismConfig::default())
+                .await
+                .unwrap();
```

Comment:
We don't need to do this now, but I think some `expects` with an appropriate message might be better here (since it's a demo). Same goes for all uses of `unwrap` here.